src/ansys/dpf/core/_custom_operators_helpers.py

```diff
@@ -70,6 +70,10 @@ def __operator_main__(operator_functor, data):
         custom_type_field.CustomTypeField,
         external_operator_api.external_operator_put_out_custom_type_field,
     ),
+    (
```

Comment:
@PProfizi is it tested somewhere else?

Reply:
@cbellot000 CodeCov says yes, which is weird because it is not tested explicitly.

Reply:
@cbellot000 BTW why do we have a `type_to_output` specific to the custom operators?

Reply:
@cbellot000 I jsut realized that to be be able to test this (and for this to even work), the changes need to be propagated to `dpf-site.zip`.
So either I add the capability to update the `dpf-site.zip` based on the current venv as we discussed previously, and use it at the start of `test_python_plugins.py`,
or we have to merge it first, then wait for a server sync loop to put it in `dpf-site.zip` and only then add the tests.

Reply:
See https://github.com/ansys/pydpf-core/pull/1379

Reply:
I see then, you can maybe add the test ad mark it as skip this way we can remove the skip later?

Reply:
> @cbellot000 BTW why do we have a `type_to_output` specific to the custom operators?

I don't understand, where would you put "external_operator_put_out_generic_data_container" instead?